octodns_constellix/__init__.py

```diff
@@ -525,36 +543,42 @@ def _data_for_pools(self, _type, records):
                     # fetch record geofilter data
                     geofilter_id = record['geolocation']['geoipFilter']
 
-                # populate rules
-                if geofilter_id == 1:
-                    rules.append({'pool': pool_name})
-                else:
+                # fetch options
+                notes = self._parse_notes(record.get('note', ''))
+
+                rule_order = notes['rule-order']
```

Comment:
Would this have a problem handling old/non-existent notes w/o the `rule-order` present? Would that throw a `KeyError` here?

Reply:
Yes, good point. I will add the necessary change tomorrow. Or you might add it if you like. :smiley: